server/User.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/*Contains the User struct, as well as any methods*/
+package main
+
+import (
+
+)
+
+//All strings for now, type will be changed as project develops
+type User struct {
+	Name string
+	ID string
+	ProfilePic string
+	Bio string
+	AlbumName string
+	Conversations []string
```

Comment:
Don't think this field should be a part of User

Reply:
It's in the schema we designed though. Where would you suggest it go instead?

Reply:
I think I mentioned this when we were discussing the design: we should just have a separate conversation entity that stores the ids of users that participate in it. Then we can just query fo conversations that contains this user

Reply:
Yes, but instead of querying every single conversation to see if the user is in it, this array is going to contain a list of Conversation IDs, so we can fetch the conversations by ID without querying for them first

Reply:
Perhaps I should give the array a better name to reflect this

Reply:
In my opinion, you don't need to reflect the user-conversation relation in the User entity at all. 
But let's leave it as is for now and return to the question when actually implementing the messaging.